usaspending_api/etl/tests/integration/test_load_to_from_delta.py

```diff
@@ -33,6 +33,24 @@
 from usaspending_api.recipient.models import RecipientLookup
 
 
+@fixture
+def populate_broker_data_to_delta():
+    # Create broker data to be imported to delta
+    dummy_broker_subaward_data = json.loads(Path("usaspending_api/awards/tests/data/broker_subawards.json").read_text())
+
+    connection = connections["data_broker"]
```

Comment:
> Probably preferable to use the usaspending_api.common.helpers.sql_helpers.get_broker_dsn_string() helper function here. When under a test, that will still surface the test_data_broker DB, yes? Since in that function it still gets the connection from django via settings.DATABASES["data_broker"].

every time I tried:
```
        with psycopg2.connect(dsn=get_broker_dsn_string()) as connection:
            with connection.cursor("usaspending_data_transfer") as cursor:
                # nuke any previous data just in case
                cursor.execute("truncate table subaward restart identity cascade;")
```
it would always crash. I couldn’t figure out why specifically. I can dig deeper into that.

Reply:
Ah I see. Took out the `"usaspending_data_transfer"` and that got it working